tools/shell-dev-commands/src/main/java/org/apache/unomi/shell/commands/ActionListCommand.java

```diff
@@ -57,14 +54,14 @@ protected Object doExecute() throws Exception {
         DataTable dataTable = new DataTable();
 
         for (ActionType actionType : allActions) {
-            ArrayList<Object> rowData = new ArrayList<Object>();
+            ArrayList<Comparable> rowData = new ArrayList<>();
```

Comment:
I left it as Comparable, since we later cast to Comparable, so this is safer IMHO.